DMX.Core.Api/Models/Devices/DeviceCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace DMX.Core.Api.Models.Devices
+{
+    public class DeviceCommand
```

Comment:
Let's distinguish these external models from our localized models.
This is not something I mentioned in the video. but here's a visualization.

```mermaid
flowchart RL
    A[External Device] --> B{{Device Foundation Service}} --> Device
```

The one thing I'm hoping you could educate me on is, how do we know which devices belong to the same NUK/Host? Is there some common ID somewhere?

Reply:
> Let's distinguish these external models from our localized models. This is not something I mentioned in the video. but here's a visualization.

Updated external models naming to prefix with `External`.

> The one thing I'm hoping you could educate me on is, how do we know which devices belong to the same NUK/Host? Is there some common ID somewhere?

Discussed in standup.